cmd/monaco/account/deploy.go

```diff
@@ -51,23 +51,20 @@ func deployCommand(fs afero.Fs) *cobra.Command {
 		Use:               "deploy <manifest.yaml> [flags]",
 		Short:             "Deploy account management resources",
 		Example:           "monaco account deploy manifest.yaml [--account <account-name-in-manifest>] [--project <project-defined-in-manifest>]",
-		Args:              cobra.ExactArgs(1),
```

Comment:
Should we use `cobra.MaximumNArgs` to still validate whether it has arguments set? could be a typo as well.